tests/indent/dart_spec.lua

```diff
@@ -11,19 +10,14 @@ local run = Runner:new(it, "tests/indent/dart", {
 describe("indent Lua:", function()
   describe("whole file:", function()
     run:whole_file(".", {
-      expected_failures = {},
+      expected_failures = {
+        "./dart/multiple_arguments.dart", -- expected failure issue #4637
+      },
     })
   end)
 end)
 
 describe("new line:", function()
   run:new_line("class.dart", { on_line = 2, text = "var x;", indent = 0 })
   run:new_line("try.dart", { on_line = 2, text = "var x;", indent = 4 })
-  run:new_line("multiple_arguments.dart", { on_line = 10, text = "var x;", indent = 4 })
-  run:new_line(
-    "multiple_arguments.dart",
-    { on_line = 11, text = "var x;", indent = 4 },
-    "expected failure issue #4637",
-    XFAIL
-  )
```

Comment:
I added it back as even though I tried putting the entire file as expected failure it would still result in a failure for the common indentation test. Not sure what approach to take here

Reply:
> I added it back as even though I tried putting the entire file as expected failure it would still result in a failure for the common indentation test. Not sure what approach to take here

Do both :D
but keep the XFAIL test, I will try to look at it later

Reply:
Added it back!